content/post/wxs-reflection-003.md

```diff
@@ -0,0 +1,61 @@
++++
+title = 'Cultivating the Natural State'
+date = 2024-06-13T12:00:00-04:00
+authors = ["Xavier Snelgrove"]
++++
+
+I skipped last week's reflection, as I was on a retreat. Before you dock me a
+few points hear me out, there's a sense in which this retreat is field research
+for my project. The leader Bonnitta Roy is an independent scholar and
+philosopher who runs an online Zoom/Substack philosophy community called [The
+Pop-Up School](https://bonnittaroy.substack.com/). This retreat, on "Cultivating
+the Natural State" was an in-person gathering in rural Vermont of primarily
+people in the the Pop-Up School's community.
+
+Roys community is interesting on many levels. She's doing work which is academic
+in flavour, but because she does not have the constraints that the conventional
+academic discipline might impose, so there is also the feeling that you're
+participating in something *new*. Something that doesn't fit into the existing
+boxes, but perhaps rhymes with them.
+
+A theme of the retreat was on "left-brain right-brain integration", and this was
+reflected in its content which would run from embodiment exercises and group
+music jamming, all the way to very verbal and diagrammatic expositions of Roy's
+ongoing metaphysics development project. Her interests span this range, and
+therefore so does the community she attracts. The attendees were striking in
+their breadth, we had people in their 20s and people in their 70s, we had German
+web developers, fourth generation Idaho herbalists, Haitian-Canadian community
+activists, academics, recovering housewives and recovering 1960s acid-heads.
+
+My father quit his second tenured academic job to enter industry in my early
+childhood (the inability to focus runs deep in this family). One thing he told
+me he appreciated when he left academia was precisely the *diversity* of folks
+he would work with. He went from leading labs of exclusively 20-30-something
+technological-neurotype keeners who had always been top of their class to being
+in orgs with Japanese salespeople and ex-military folk and future librarians and
+technical writers and marketing gurus and alcoholic entrepreneurs.
+
+Sons are cursed to rebel against their fathers while ultimately just playing out
```

Comment:
I've also been thinking about my father this week (probably we're both feeling the ambient vibes of fathers day).

When my dad moved out to Silicon Valley in his 50's he told me that he finally experienced the dream of socialism that they kept promising him in his Soviet childhood - at least inside the corporation it felt like the socialist utopia - freely available food and drink everywhere, corporate wellness deparment, activities, etc. 

The coddling / never-ending childhood I associate with SV culture he felt as what he was promised as socialism!